control_panel_api/models.py

```diff
@@ -60,6 +62,18 @@ def aws_delete_role(self):
         services.delete_role(self.aws_role_name)
 
 
+class AppUser(TimeStampedModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
I think we should be consistent with the other "association models" and add `related_name='userapps'` here and to the `app` association below.